lume_model/base.py

```diff
@@ -347,14 +353,15 @@ def yaml(
             base_key: str = "",
             file_prefix: str = "",
             save_models: bool = False,
+            save_jit: bool = False,
     ) -> str:
         """Serializes the object and returns a YAML formatted string defining the model.
 
         Args:
             base_key: Base key for serialization.
             file_prefix: Prefix for generated filenames.
             save_models: Determines whether models are saved to file.
-
+            save_jit: Determines whether the structure of the model is saved as TorchScript
         Returns:
```

Comment:
Does the body of this function use the `save_jit` input argument?

Reply:
its does now :)